proposals/53:kread-start/mn2-start.test.js

```diff
@@ -0,0 +1,441 @@
+// @ts-check
```

Comment:
consider noting that the reason for the extreme Ava timeout is there's a lot of work done here.

Going forward I think it behooves us to have a straightforward script like `performActions.ts`

Reply:
sure... if the code weren't already working, I would not likely approach it this way.

Though I did do a little bit of `performActions.ts` style work way back in the beginning, and since switching to `test()` style, I have gotten a lot of mileage out of being able to run just selected tests.

Reply:
ah. mis-read at first. you're suggesting adding a comment. got it.

Reply:
> gotten a lot of mileage out of being able to run just selected tests

That makes sense for tests! If it's really testing something that doesn't depend on anything happening, then I think that makes sense in Ava. E.g. that bundles aren't already installed. But that can be in `yarn ava pre.test.js` in `eval.sh` before the action is performed.